### ISSUE TITLE
Sign-in sheet: Bottom/global "Done" button fails when adding new attendee

### ISSUE THREAD

**Issue by cecilia-donnelly** [OpenTechStrategies/swoptact#127]
To reproduce: 
Click on "Demo Event."  Scroll down, click "Add New" and type "sim" then arrow down, select "Simple Addition" and tab over to the next field.  Click the local "Done" button in that row.  Click the global "Done" button on the bottom right of the page.  Nothing happens (it should return the user to the list of Actions).

Looking at the console, it shows that the problem is in `inline_ajax.js`, line 617.  `text_inputs[0]` isn't defined, probably because we've stopped using the `vTextField` class referenced in line 614.  (just my guess).  That causes the `updateInlinedModel()` function to fail, and so clicking the global "Done" fails silently.

**Comment by cecilia-donnelly** [comment]
@cwebber here's the diff that fixes this:
```
--- a/swoptact/static/js/inline_ajax.js
+++ b/swoptact/static/js/inline_ajax.js
@@ -773,11 +773,6 @@ function setupInlinedModelCallbacks() {
         function(event) {
             event.preventDefault();
             addNewInlinedModel();
-            // addNewInlinedModel saves all open rows, so we empty the existing
-            // array of rows being edited.
-            rows_editing = [];
-            // this case still needs to be handled correctly
-            rows_editing.push('empty');
         });
 
     $("#inlined-model-table").on(
```

As far as I can tell, we were manually adding "empty" to the list of rows that needed to be saved, but the new row doesn't actually have the class for an empty model, thanks to some autocomplete magic.  So, when the global "Done" button tries to go through the "list of rows that need to be saved" (`rows_editing`), it was looking for one with the id `inlined-model-edit-` but wasn't finding it, because we had somewhere already appended the Participant ID to that row id along the way.

Removing the lines above fixed this error for all the cases I tried.

**Comment by cwebber** [comment]
Well, see also #105... this rows_editing stuff is broken anyway so we probably scrap it and go with an approach that doesn't involve tracking duplicate state or risk race conditions anyhow.

**Comment by cecilia-donnelly** [comment]
Sure, that sounds good to me.

**Comment by cwebber** [comment]
This was fixed via #105
